source/develop/release-management/features/network/autodefine-external-network.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Autodefine External Network
+category: feature
+authors: almusil
+feature_name: Autodefine External Network
+feature_modules: engine
+feature_status: In Development
+---
+
+# Autodefine External Network
+
+## Summary
+
+This feature extends [Provider Physical Network](https://www.ovirt.org/develop/release-management/features/network/provider-physical-network/) 
+adding support for auto-defined external network connected to physical ones. 
+Engine automatically creates external network based on physical VM network with 
+physical connection to it. 
+
+Related patches can be found on [gerrit topic:autodefine](https://gerrit.ovirt.org/#/q/topic:autodefine).
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+### Benefit to oVirt
+
+Autodefine helps with reducing administrative tasks that needs to done with OVS 
+switch type in the cluster.
+
+## Prerequisites
+
+Cluster has to be configured with switch type OVS and valid external provider.
+The user needs to make sure that connection to external provider is valid.
+
+### Limitations
+
+* With OVS switch type in cluster only external networks are allowed to 
+  be attached to the VM interface. 
+
+* The user cannot edit or remove auto-defined external network.
```

Comment:
Please refer to the comments in -
https://trello.com/c/ol5tzdOH/12-auto-define-an-ovn-localnet-on-ovs-vm-network